website/templates/project/contributors.mako

```diff
@@ -353,8 +356,10 @@
 
                 <!-- ko ifnot: contributor.canEdit() -->
                     <!-- ko if: canRemove -->
```

Comment:
Well, the funky `canEdit`/`canRemove` was present before this PR, and it looks like the permissions code in `contribManager.js` is a [known issue](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/static/js/contribManager.js#L35-L36). Reticketed a refactor as [OSF-5531](https://openscience.atlassian.net/browse/OSF-5531).